### ISSUE TITLE
Fine-tuning in Calvin

### ISSUE THREAD

**Issue by yzzdz** [baaivision/UniVLA#14]
I hope you're doing well! I have been following the instructions in the README for fine-tuning WORLD_MODEL_POSTTRAIN data in Calvin-ABCD. However, I’ve encountered an issue where the loss is decreasing  slowly during training.

Could you kindly provide the training loss logs? This would help me better understand the results and make appropriate adjustments to improve the training process.

**Comment by Robertwyq** [comment]
@yzzdz I guess you mean finetune for Calvin performance? or train the world model for video generation？
The training log is already provide in the ckpt folder. https://huggingface.co/Yuqi1997/UniVLA/blob/main/UNIVLA_CALVIN_ABCD_VIDEO_BS192_8K/trainer_state.json
